lib/Rename/Adapter/ClassMover/FileRenamer.php

```diff
@@ -66,8 +66,8 @@ public function renameFile(TextDocumentUri $from, TextDocumentUri $to): Promise
                 }
             }
 
-            return new WorkspaceRenameEdits([
-                LocatedTextEditsMap::fromLocatedEdits($locatedEdits),
+            return new WorkspaceOperations([
+                LocatedTextEditsMap::fromLocatedEdits($locatedEdits)->toLocatedTextEdits(),
```

Comment:
```suggestion
                ...LocatedTextEditsMap::fromLocatedEdits($locatedEdits)->toLocatedTextEdits(),
```
Missing unpack here.